codex/blockexchange/network/networkpeer.nim

```diff
@@ -72,6 +74,8 @@ proc broadcast*(b: NetworkPeer, msg: Message) =
   proc sendAwaiter() {.async.} =
     try:
       await b.send(msg)
+    except CancelledError as error:
+      raise error
```

Comment:
Don't re-raise in a proc that is `asyncSpawn` it will be terminate the application because it is converted to a `Defect`. Anything that is `asyncSpawn` should not raise exceptions.

Reply:
Good catch, I misread the `except` as being part of `broadcast` instead of `sendAwaiter`.

By the way, it seems that `broadcast` is unused? Should we remove it?